airflow/providers/google/cloud/hooks/kubernetes_engine.py

```diff
@@ -507,33 +507,37 @@ async def get_pod(self, name: str, namespace: str) -> V1Pod:
         :param name: Name of the pod.
         :param namespace: Name of the pod's namespace.
         """
-        async with Token(scopes=self.scopes) as token:
-            async with self.get_conn(token) as connection:
-                v1_api = async_client.CoreV1Api(connection)
-                pod: V1Pod = await v1_api.read_namespaced_pod(
-                    name=name,
-                    namespace=namespace,
-                )
-            return pod
+        async with self.service_file_as_context() as service_file:  # type: ignore[attr-defined]
```

Comment:
is there any change in the default auth behavior which might be customers need to be aware of?

Reply:
I don't think there's any change needed. I just updated the PR description and mentioned it